app/src/ui/octicons/octicon.tsx

```diff
@@ -34,14 +36,34 @@ interface IOcticonProps {
  * which is why the width and height properties specify the maximum and
  * not the minimum size.
  *
- * Usage: `<Octicon symbol={OcticonSymbol.mark_github} />`
+ * Usage: `<Octicon symbol={octicons.markGithub} />`
  */
 export class Octicon extends React.Component<IOcticonProps, {}> {
   private svgRef = createObservableRef<SVGSVGElement>()
 
   public render() {
-    const { symbol, title, tooltipDirection } = this.props
-    const viewBox = `0 0 ${symbol.w} ${symbol.h}`
+    const { symbol } = this.props
+
+    if (this.isSingleVariant(symbol)) {
+      return this.renderIcon(symbol.p, symbol.h, symbol.w)
+    } else {
+      const height = this.props.height ?? 16
+      const naturalHeight = this.closestNaturalHeight(
+        Object.keys(symbol).map(h => parseInt(h, 10)) as Array<number>,
+        height
+      )
+
+      const scaledSymbol = symbol[naturalHeight]!
```

Comment:
```suggestion
      const scaledSymbol = symbol[naturalHeight]
      if(scaledSymbol === undefined) {
           // Just a type guard, shouldn't happen as we are getting the height from the symbol object.
          return null
      }
```

We avoid using the type assertion in favor of type guards since type assertion bypassing the purpose of using typescript. Type guards let us intentionally define how we want to handle the "should be impossible" case. Here, I think a comment and a return null is fine -> but others it might make sense to do a non fatal error to help debug. But, I think if we ever had an instance of a Octicon not showing up, we would look at the Octicon file. :P